.circleci/config.yml

```diff
@@ -74,13 +74,22 @@ jobs:
 
   test-firefox:
     docker:
-      - image: circleci/node:latest
+      - image: circleci/node:latest-browsers
     steps:
       - attach_workspace:
           at: ~/
+      - run:
+          name: Install the latest Firefox
```

Comment:
If we need to install the latest Firefox, do we need to do the same with Chrome?